test/specs/happy.js

```diff
@@ -604,6 +604,18 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       welcome.clickOnOpenModalButton()
       welcome.verifyTitle(welcomeCopy)
     })
+
+    it('should be able to close modal with ESC button', async () => {
+      driver.get(localhostUrl + `?useModal=true`)
+      const welcomeCopy = welcome.copy()
+      welcome.clickOnOpenModalButton()
+      welcome.verifyTitle(welcomeCopy)
+      driver.sleep(500)
+      welcome.pressEscapeButton()
+      driver.sleep(500)
+      welcome.clickOnOpenModalButton()
+      welcome.verifyTitle(welcomeCopy)
+    })
```

Comment:
This test and the other modal open/close modal test in lines 596 - 606 are identical aside from whether it is a UI button click or `Esc` button keypress code would be more maintainable with a common helper function.

Reply:
Do you mean to create a function that could be reused and include it in the common class not welcome one? 

Sth like

```
async pressKeyboardButton(button) {
    this.title.sendKeys(Key.button)
  }
```

Reply:
Done in: 48df903